src/signpdf.test.js

```diff
@@ -163,18 +167,58 @@ describe('Test signing', () => {
         expect(signedData instanceof Buffer).toBe(true);
     });
     it('signs a ready pdf two times', async () => {
+        const p12Buffer = fs.readFileSync(`${__dirname}/../resources/withpass.p12`);
+        let pdfBuffer = fs.readFileSync(`${__dirname}/../resources/w3dummy.pdf`);
+        pdfBuffer = plainAddPlaceholder({
+          pdfBuffer, 
+          reason: 'first',
+          signatureLength: p12Buffer.length
+        });
+        pdfBuffer = signer.sign(pdfBuffer, p12Buffer, { passphrase: 'node-signpdf' });
         const secondP12Buffer = fs.readFileSync(`${__dirname}/../resources/withpass.p12`);
         let signedPdfBuffer = fs.readFileSync(`${__dirname}/../resources/signed-once.pdf`);
-        signedPdfBuffer = plainAddPlaceholder({
-            pdfBuffer: signedPdfBuffer,
-            reason: 'second',
-            signatureLength: 1592,
+        signedPdfBuffer = plainAddPlaceholder({ 
+          pdfBuffer: signedPdfBuffer, 
+          reason: 'second',
+          signatureLength: secondP12Buffer.length
```

Comment:
The length of the p12 buffer does not relate to the length of the signature. Read the note here: https://github.com/vbuch/node-signpdf/tree/6d74d98d2038837d29055f346be7dd6b758158ac#append-a-signature-placeholder